.trivyignore

```diff
@@ -0,0 +1,17 @@
+# The CyberArk Google Marketplace App deployer container is based upon
+# the Google deployer_helm:0.10.1 container image, and is subject to
+# the following vulnerabiliities:
+CVE-2019-10220
+CVE-2019-19813
+CVE-2019-19814
+CVE-2019-19816
+
+# The postgres:9.4 container is subject to the following critical
+# vulnerabilities:
+CVE-2005-2541
+CVE-2017-1000082
+
+# The nginx:1.17 container is subject to the following critical
+# vulnerabilities:
+CVE-2005-2541
+CVE-2019-2201
```

Comment:
I think we need to specifically have a comment line for each of these so they're easy to identify without googling